wrims-core/build.gradle

```diff
@@ -60,21 +62,6 @@ dependencies {
 
 }
 
-configurations.all {
-    exclude group: "javax.media", module: "jai_core"
-    exclude group: "javax.media", module: "jai_codec"
-    exclude group: "com.sun.media", module: "jai_imageio"
-}
-
-tasks.register('copyDependencies', Copy) {
```

Comment:
Was this meant to be removed?

Reply:
Yes, this was a deliberate cleanup in the build.